pyccel/codegen/compiling/basic.py

```diff
@@ -238,9 +253,17 @@ def release_simple_lock(self):
 
     @property
     def accelerators(self):
-        """ Returns the names of the accelerators required to compile the file
         """
-        return self._accelerators
+        Get the names of the accelerators required to compile the file.
+
+        Return a set containing the name of all accelarators required
+        to compile the file. An accelerator is a tool used to add a new
+        capacity to the code. Such an addition requires multiple flags
+        (includes/libs/libdirs/etc) and is therefore specified separately
+        in the compiler configuration file. Examples of 'accelerators' are:
+        openmp, openacc, python.
```

Comment:
Mostly because "accelerator" is a bad choice of name. We chose "accelerators" when we were only concerned with mpi/openmp. However in order to link python we need includes, and libs, libdirs or directories. These are all platform dependant so it makes the most sense to add them as a sub-category in the compilers dictionary. This property returns those subdictionaries.

Reply:
Luckily these are Pyccel internals 😄